bundles/ApplicationLoggerBundle/src/Controller/LogController.php

```diff
@@ -47,11 +48,7 @@ public function onKernelControllerEvent(ControllerEvent $event): void
         }
     }
 
-    /**
-     * @Route("/log/show", name="pimcore_admin_bundle_applicationlogger_log_show", methods={"GET", "POST"})
-     *
-     *
-     */
+    #[Route(path: '/log/show', name: 'show', methods: ['GET', 'POST'])]
```

Comment:
With the new attribute on the class, the paths should now be `/log/log/show`, which is not correct, I think.

Reply:
Nevermind, you already fixed it :D

Reply:
Yes. I've fixed it after self-review. Thanks for being vigilant :)